packages/Textarea/src/mixins.scss

```diff
@@ -0,0 +1,25 @@
+@use '~@igloo-ui/tokens/dist/base10/variables' as tokens;
+
+@mixin fancy-webkit-scrollbar(
+  $background-color: #{tokens.$samoyed},
+  $scrollbar-color: #{tokens.$grey-300},
+  $scrollbar-width: 1rem) {
+  -webkit-overflow-scrolling: touch;
+  scrollbar-width: thin;
+
+  &::-webkit-scrollbar {
+    width: $scrollbar-width;
+    height: $scrollbar-width;
+    background-color: transparent;
+  }
+
+  &::-webkit-scrollbar-thumb {
+    background-color: $scrollbar-color;
+    border: 0.2rem solid $background-color;
```

Comment:
Did you reversed `$scrollbar-color` and `$background-color` here?

Reply:
It should be like that but maybe I'll rename the variables to make more sense. I took this from the old component, but I can improve it, thanks.